cpm

```diff
@@ -162,7 +161,7 @@ su_do() {
   if [ "$(id -u)" != 0 ]; then
     if command -v sudo >/dev/null; then
       sudo "$@"
-    elif command -v doas >/dev/null && [ -f /etc/doas.conf ]; then
+    elif command -v doas >/dev/null && [ -f /etc/doas.conf ] || [ -f /usr/local/etc/doas.conf ]; then
```

Comment:
The logic here is broken. While this is unlikely to be an issue, this is equivalent to: `if ((a && b) || c)`, so `doas` does not need to exist for this to attempt to use it. Please add `()` around the `||` clause.